_articles/github-for-collaboration/collaborative-workflow-for-contributions.md

```diff
@@ -60,6 +60,10 @@ When you’re writing commit messages, you’ll want to make them short, clear,
     4.  Write a great commit message
     5.  Make a pull request!
 
+![New Pull Request]({{ site.baseurl }}/img/create-pull-req-1.png)
```

Comment:
Sorry @nufail - looks like I didn't look close enough! Yes, I think @betatim's suggest would work well. At least, having some captioning for each image would help.